src/components/Loading/Loading.tsx

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled, { css } from 'styled-components';
+
+import { Padbox, Stack } from '../layout';
+import { ProgressBar } from '../ProgressBar';
+import { Spinner } from '../Spinner';
+import { Text } from '../typographyLegacy';
+import { pxToRem } from '../../utils';
+import { LoadingRootProps } from './Loading.types';
+
+const LoadingRoot = styled(Padbox)<LoadingRootProps>`
+  display: flex;
+  align-items: center;
+  ${({ $height }) => css`
+    min-height: ${pxToRem($height)};
+  `};
+`;
+
+export const Loading = ({ progress, subject, message, testId, height }) => {
```

Comment:
```suggestion
const Loading = ({ progress, subject, message, testId, height }) => {
```

I would drop the named export and add `export default Loading` to the end of file, just for consistency sake